### PR TITLE
howard-bc: update to 1.1.1

### PR DESCRIPTION
**Pull request by gavinhoward**
I have released a new version with history support and much more.

There are two releases in this update: [1.1.0][1] and [1.1.1][2].

The biggest changes to the package are updates to the build system and allowing the test suite to run. (I added a mode to ignore tests that needed to be generated by an existing `bc`.)

[1]: https://github.com/gavinhoward/bc/releases/tag/1.1.0
[2]: https://github.com/gavinhoward/bc/releases/tag/1.1.1

### CODE REVIEW

**Comment by gavinhoward** [comment]
Going to close until I know what is going on.

File: testing/howard-bc/APKBUILD [hunk 1]
```diff
@@ -2,28 +2,29 @@
...
-options="!check"
 
 build() {
 	cd "$builddir"
-	CFLAGS="-flto" make release
+	PREFIX=/usr DESTDIR="$pkgdir" EXECSUFFIX=-howard ./configure.sh -GO3
+	make
+}
+
+test() {
```
Comment (andypost) [inline testing/howard-bc/APKBUILD]: it should be `check()`

Comment (gavinhoward) [reply testing/howard-bc/APKBUILD]: Okay.

Comment (andypost) [reply testing/howard-bc/APKBUILD]: use `abuild checksum` to update checksum

Comment (gavinhoward) [reply testing/howard-bc/APKBUILD]: I have been doing that. Am I supposed to do that *and* commit the change?

Comment (andypost) [reply testing/howard-bc/APKBUILD]: yep, checksums are required to check downloaded data

**Pull request closed without merging**